portfolio/src/main/webapp/script.js

```diff
@@ -30,3 +30,11 @@ function addRandomQuote() {
   const quoteContainer = document.getElementById('quote-container');
   quoteContainer.innerText = quote;
 }
+
+console.log("hello world");
```

Comment:
Remove console.log() statement before merging.

Reply:
got it!